web-services/client/src/main/java/datawave/webservice/model/ModelList.java

```diff
@@ -55,7 +72,8 @@ public String getPageHeader() {
      */
     @Override
     public String getHeadContent() {
-        return EMPTY;
+        String dataTablesHeader = MessageFormat.format(DATA_TABLES_TEMPLATE, jqueryUri, dataTablesUri);
```

Comment:
Get rid of the unused local variable and change this to `return MessageFormat.format(DATA_TABLES_TEMPLATE, jqueryUri, dataTablesUri);` please.